### PR TITLE
Upload docker compose file

### PR DESCRIPTION
**Pull request by shaners1106**

### CODE REVIEW

**Comment by joseph-flinn** [comment]
@shaners1106 Can you create a `foodtruck` directory in the `sprint-2` directory and move the `docker-compose.yml` file there?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+version: "3"
+services:
+  es:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.2
```
Comment (joseph-flinn) [inline docker-compose.yml]: This version tag needs to be `7.8.0` or higher in order to pull the `ARM` version

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    container_name: es
+    environment:
+      - discovery.type=single-node
+    ports:
+      - 9200:9200
+    volumes:
+      - esdata1:/usr/share/elasticsearch/data
+  web:
+    image: prakhar1989/foodtrucks-web
```
Comment (joseph-flinn) [inline docker-compose.yml]: This is going to have to be a manually built docker image on your machine (like the `static-site` one). The git repo with the Dockerfile should be linked in the Tutorial. Whatever you tag that image as should go here.

**Comment by shaners1106** [comment]
When you get a second, can you take a look at the newly edited docker-compose.yml file to make sure I made accurate changes?  I tagged that image with  shaners1106/foodtrucks

Thank you Joseph!

**Comment by joseph-flinn** [comment]
@shaners1106 The version at the top of the docker-compose file is the version of `docker-compose` that you need to use. That actually needs to be `3` and not `7.8.0`. I was talking about the `6.3.2` version of the elasticsearch image: `elasticsearch:6.3.2`. This instead needs to be `elasticsearch:7.8.0`.

It looks like you edit for the food trucks image is also missing. The docker-compose file is still using the `prakhar1989/foodtrucks-web` image.

**Comment by shaners1106** [comment]
I did make those last few changes to the docker-compose.yml file

**Comment by joseph-flinn** [comment]
Ah, I didn't see that. There's a little circular arrow up near my avatar at the top that will re-request my review once you have updated a PR. That will notify me that there is something for me to review. Also, feel free to add me as a reviewer when you open your PR

image

**Review by joseph-flinn — APPROVED** [review]

**Comment by shaners1106** [comment]
Gotcha, I did not know that.  Good to know!  It's crazy how much I didn't
know about Github...

**Comment by joseph-flinn** [comment]
@shaners1106 No worries at all! That's what this is for. Most of this is what you would learn on the job in an internship. I think schools could be a lot better at teaching version control, but they really lean on the internships to do so. But then again, maybe version control practices in industry has changed a bunch since the profs have been there.

That being said, I am still learning `git` and `GitHub` too and some companies and teams might do things a bit different than others. So it'll probably be a career/life long learning process.

**Pull request merged**